tools/downloader/topologies.py

```diff
@@ -0,0 +1,13 @@
+import os
+
+from pathlib import Path
+
+import sys
+import common
+import downloader
+import converter
+
+_cache = Path(os.getenv('OPEN_MODEL_ZOO_CACHE_DIR', Path('.').resolve() / 'models'))
+
+def load(model_name):
+    downloader.main(['--name=' + model_name, '--output_dir=' + str(_cache).replace('\\', '\\\\')])
```

Comment:
Let's create `Zoo` class which can provide some context and have supportive methods (i.e. list all the models names). So method can be named as `load_model` (see https://github.com/openvinotoolkit/open_model_zoo/pull/304#issuecomment-567436663)

Reply:
please ignore it